### PR TITLE
Using logic java robot.

### PR DESCRIPTION
**Pull request by EliosPride**

### CODE REVIEW

File: src/main/java/core/basesyntax/FieldXY.java [hunk 1]
```diff
@@ -12,8 +12,27 @@
  */
 public class FieldXY {
 
-    public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+    public void moveRobot(Robot robot, int coordinateX, int coordinateY) {
+        robot.getCoordinateX();
+        robot.getCoordinateY();
+        robot.getDirection();
```
Comment (al-pavlenko) [inline src/main/java/core/basesyntax/FieldXY.java]: @EliosPride What is this?

File: src/main/java/core/basesyntax/FieldXY.java [hunk 2]
```diff
@@ -12,8 +12,27 @@
...
+        int[] toX = new int[coordinateX - robot.getCoordinateX()];
+        int[] toY = new int[coordinateY - robot.getCoordinateY()];
```
Comment (al-pavlenko) [inline src/main/java/core/basesyntax/FieldXY.java]: @EliosPride you can get here exeption if result is negative

File: src/main/java/core/basesyntax/FieldXY.java [hunk 3]
```diff
@@ -12,8 +12,27 @@
...
+            }
+            for (int a = 0; a < toY.length; a++) {
+                if (coordinateY - (robot.getCoordinateY() + 1) > coordinateY - robot.getCoordinateY()) {
+                    robot.turnRight();
+                }
+                if (coordinateY - (robot.getCoordinateY() + 1) < coordinateY - robot.getCoordinateY()) {
+                    robot.stepForward();
+                }
+            }
+        }
```
Comment (al-pavlenko) [inline src/main/java/core/basesyntax/FieldXY.java]: @EliosPride you don't need this complexity, split moves to separate loops

**Comment by al-pavlenko** [comment]
@EliosPride Also add Main class with logic testing

File: src/main/java/core/basesyntax/FieldXY.java (hunk 1 above)
Comment (EliosPride) [reply src/main/java/core/basesyntax/FieldXY.java]: @al-pavlenko done

Comment (EliosPride) [reply src/main/java/core/basesyntax/FieldXY.java]: @al-pavlenko delete rudiment

File: src/main/java/core/basesyntax/FieldXY.java (hunk 2 above)
Comment (EliosPride) [reply src/main/java/core/basesyntax/FieldXY.java]: @al-pavlenko correct mistake, willn't have any exception

**Pull request merged**